### ISSUE TITLE
enable cron jobs for desktop environment

### ISSUE THREAD

**Issue by helenkirk** [opencadc/science-platform#315]
This is a follow-up to the now closed issue 'update leap second table for CASA #33' (https://github.com/opencadc/skaha/issues/33).  The short-term solution to resolve the leap second table in that issue is for me to periodically run an rsync command in a desktop session.  Longer-term, it is more sustainable to set up this command as a cron job, but my understanding is that some backend development needs to happen for cron jobs to be able to be implemented.

**Comment by sfabbro** [comment]
@brianmajor this could be a good feature to have user cron job and possibly already natively supported by kubernetes? A use case such as synchronisation of calibration files is a good one, but likely other will pop up.

**Comment by brianmajor** [comment]
Yes, I agree, that would be the way to add cron support.  We might find
some creative uses of that.

**Comment by leungcalvin** [comment]
+1 for wanting some sort of persistent session from which cronjobs can be run! We want to use a similar workflow for auto-processing CHIME data.
